LanguageFeatures/Inline-classes/syntax_A02_t13.dart

```diff
@@ -0,0 +1,285 @@
+  AbstractInterfaceInlineClass2(this.id);
+}
+
+inline abstract interface class AbstractInterfaceInlineClass3 {
+//     ^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractInterfaceInlineClass3(this.id);
+}
+
+abstract final inline class AbstractFinalInlineClass1 {
+//             ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractFinalInlineClass1(this.id);
+}
+
+abstract inline final class AbstractFinalInlineClass2 {
+//       ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractFinalInlineClass2(this.id);
+}
+
+inline abstract final class AbstractFinalInlineClass3 {
+//     ^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractFinalInlineClass3(this.id);
+}
+
+
+mixin inline class MixinInlineClass1 {
+//    ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  MixinInlineClass1(this.id);
+}
+
+inline mixin class MixinInlineClass2 {
+//     ^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  MixinInlineClass2(this.id);
+}
+
+base mixin inline class BaseMixinInlineClass1 {
+//         ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  BaseMixinInlineClass(this.id);
+}
+
+base inline mixin class BaseMixinInlineClass2 {
+//   ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  BaseMixinInlineClass2(this.id);
+}
+
+inline base mixin class BaseMixinInlineClass3 {
+//     ^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  BaseMixinInlineClass3(this.id);
+}
+
+abstract mixin inline class AbstractMixinInlineClass1 {
+//             ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractMixinInlineClass1(this.id);
+}
+
+abstract inline mixin class AbstractMixinInlineClass2 {
+//       ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractMixinInlineClass2(this.id);
+}
+
+inline abstract mixin class AbstractMixinInlineClass3 {
+//     ^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractMixinInlineClass3(this.id);
+}
+
+abstract base mixin inline class AbstractBaseMixinInlineClass1 {
+//                  ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractBaseMixinInlineClass1(this.id);
+}
+
+abstract base inline mixin class AbstractBaseMixinInlineClass2 {
+//            ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractBaseMixinInlineClass2(this.id);
+}
+
+abstract inline base mixin class AbstractBaseMixinInlineClass3 {
+//       ^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractBaseMixinInlineClass3(this.id);
+}
+
+inline abstract base mixin class AbstractBaseMixinInlineClass4 {
+//     ^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+  final int id;
+  AbstractBaseMixinInlineClass4(this.id);
+}
```

Comment:
Yep! This library yields a very nice coverage of potential parser issues (a la "oh, nobody tried _that_ before!" ;-).